tests/SprykerTest/Client/RabbitMq/Model/Connection/ConnectionBuilderTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
```

Comment:
1. https://github.com/spryker/rabbit-mq/pull/77#pullrequestreview-1980694965 - not fully fixed, see https://spryker.atlassian.net/wiki/spaces/Framework/pages/3580395659/Common+CR+issues.
2. https://github.com/spryker/rabbit-mq/pull/77#discussion_r1552136007 not fixed.
3. https://github.com/spryker/rabbit-mq/pull/77#discussion_r1552140030 not fixed. (Please move `expectException` to the main method, and get rid of `//Assert/Arrange` annotations in the child method)
4.